docs/user-guide/secrets/index.md

```diff
@@ -337,7 +341,8 @@ permission for different files like this:
 ```
 
 In this case, the file resulting in `/etc/foo/my-group/my-username` will have
```

Comment:
"In this case, the resulting file in `etc/foo/my-group/my-username` will have a permission value of `0777`. Owing to JSON limitations, you must specify the mode in decimal notation."